code/modules/antagonists/_common/antag_spawner.dm

```diff
@@ -153,15 +156,18 @@
 	var/mob/living/carbon/human/M = new/mob/living/carbon/human(T)
 	C.prefs.copy_to(M)
 	M.key = C.key
+	var/datum/mind/clown_mind = M.mind
 
-	var/datum/antagonist/nukeop/clownop/new_op = new /datum/antagonist/nukeop/clownop()
+	var/datum/antagonist/nukeop/clownop/new_op = new()
```

Comment:
no sense duplicating this code if the only thing different is a single path and a name. just make them vars on the parent spawner. that way you can easily add more in the future.